source/upgrade-guide/troubleshooting.rst

```diff
@@ -0,0 +1,73 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+   :description: This section collects common issues that might occur when upgrading central components and provides steps to resolve them.
+
+
+Troubleshooting
+===============
+
+This section collects common issues that might occur when upgrading central components and provides steps to resolve them.
+
+The 'vulnerability-detector' configuration is deprecated
+--------------------------------------------------------
+
+This warning appears because when upgrading the Wazuh manager, the ``/var/ossec/etc/ossec.conf`` file remains unchanged, retaining the previous configuration of the Vulnerability Detection module. In addition, invalid configuration warnings might appear for the ``interval``, ``min_full_scan_interval``, ``run_on_start`` and ``provider`` elements. To resolve this issue, update the configuration as specified in the :doc:`Vulnerability detection documentation </user-manual/capabilities/vulnerability-detection/configuring-scans>`.
```

Comment:
```suggestion
This warning appears because when upgrading the Wazuh manager, the ``/var/ossec/etc/ossec.conf`` file remains unchanged, retaining the previous configuration of the Vulnerability Detection module. In addition, invalid configuration warnings might appear for the ``interval``, ``min_full_scan_interval``, ``run_on_start`` and ``provider`` elements. To resolve this issue, update the configuration as specified in :doc:`/user-manual/capabilities/vulnerability-detection/configuring-scans`.
```